roles/generate-jenkins/templates/README_SNIPPETS/HWACCEL.j2

```diff
@@ -16,7 +16,7 @@ We will automatically ensure the abc user inside of the container has the proper
 #### Nvidia
 
 Hardware acceleration users for Nvidia will need to install the container runtime provided by Nvidia on their host, instructions can be found here:
-https://github.com/NVIDIA/nvidia-docker
+https://github.com/NVIDIA/nvidia-container-toolkit
 
 We automatically add the necessary environment variable that will utilise all the features available on a GPU on the host. Once nvidia-docker is installed on your host you will need to re/create the docker container with the nvidia container runtime `--runtime=nvidia` and add an environment variable `-e NVIDIA_VISIBLE_DEVICES=all` (can also be set to a specific gpu's UUID, this can be discovered by running `nvidia-smi --query-gpu=gpu_name,gpu_uuid --format=csv` ). NVIDIA automatically mounts the GPU and drivers from your host into the container.
```

Comment:
`Once nvidia-docker is installed on your host` should probably read `Once nvidia-container-toolkit is installed on your host`